### ISSUE TITLE
Horizontal layout

### ISSUE THREAD

**Issue by cweiske** [madsen/vbindiff#16]
It would be nice if there was an option for horizontal layout, with one file on the left, the other on the right side. This makes sense because most displays are wide instead of high.

Currently the right side of the screen is black and unused.

(basically like `binwalk -W file1 file2 | less -R`, but more efficient)

**Comment by megaslowpoke** [comment]
Yes, and I would very much like to be able to swap the "hex" and "ascii" fields for a more visual view.
For example, how here, or vice versa, to rearrange the "hex" fields closer to the center: 
![VBinDiff_new](https://user-images.githubusercontent.com/22065152/119711419-35989380-be68-11eb-96d2-a191efe329eb.png)
